src/View/mainpage/StructureTab.py

```diff
@@ -141,12 +141,18 @@ def update_content(self):
 
 		self.scroll_area.setWidget(self.scroll_area_content)
 
-	def structure_modified(self, new_dataset):
+	def structure_modified(self, changes):
 		"""
 		Executes when a structure is renamed/deleted. Displays indicator that structure has changed.
+		changes is a tuple of (new_dataset, description_of_changes)
+		description_of_changes follows the format {"type_of_change": value_of_change}.
```

Comment:
and are of type
``dict`` with key (type of change) having type ``str`` and value (value_of_change) having type ``list`` of ``str``
And the type_of_change drives how the list is to be interpreted.
While I figured it out, it took a bit.  The examples are great.  But it is nice to have a straight up specification of the data type involved, especially given that python is loosely typed and late binding.